Makefile.am

```diff
@@ -221,7 +221,7 @@ icons_sources = $(top_srcdir)/icons/*
 
 script_sources = bootstrap.sh
 
-man_sources = $(wildcard docs/profanity*.1)
+man1_sources = $(wildcard $(top_srcdir)/docs/*.1)
```

Comment:
please mention in this commit message the commit that introduced the mistake. some explanation :)

Reply:
better now? :joy:

Reply:
yes :)